webdriver/tests/classic/delete_session/delete.py

```diff
@@ -9,27 +9,39 @@ def delete_session(session):
 
 
 def test_null_response_value(session):
-    response = delete_session(session)
-    value = assert_success(response)
-    assert value is None
+    try:
+        response = delete_session(session)
+        value = assert_success(response)
+        assert value is None
 
-    # Need an explicit call to session.end() to notify the test harness
-    # that a new session needs to be created for subsequent tests.
-    session.end()
+    finally:
+        response = session.transport.send(
+            "GET", f"session/{session.session_id}/alert/text"
+        )
+        assert_error(response, "invalid session id")
```

Comment:
Similar to the other test this should be in the `try` block, right? Otherwise we wouldn't correctly end the session in case of the assertion fails.